docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc

```diff
@@ -104,13 +100,10 @@ listen ceph_rgw
 7: br-ex	inet 10.0.0.106/24 brd 10.0.0.255 scope global br-ex\   	valid_lft forever preferred_lft forever
 ...
 ----
-+
-This example shows that `controller-0` is exposing the services by using the external network, which is not present in
-the {Ceph} nodes, and you need to propagate it through {OpenStackPreviousInstaller}.
 
-. Propagate the HAProxy frontend network to {CephCluster} nodes.
+. Propagate the HAProxy front-end network to {CephCluster} nodes.
 
-.. Change the NIC template used to define the `ceph-storage` network interfaces and add the new config section:
+.. Change the NIC template that is used to define the `ceph-storage` network interfaces and add the new config section:
```

Comment:
Could also say "Change the NIC template that you use to define the `ceph-storage` network interfaces and add the new config section:" to make it less passive unless this statement is now not true